src/components/Interpretations/common/Interpretation/InterpretationUpdateForm.js

```diff
@@ -48,7 +49,9 @@ export const InterpretationUpdateForm = ({
                     disabled={loading}
                     errorText={errorText}
                 />
-                <InterpretationSharingLink id={id} type="interpretation" />
+                {showSharingLink && (
+                    <InterpretationSharingLink id={id} type="interpretation" />
```

Comment:
I'm kind of confused about the `type` property. This gets passed to SharingDialog, right? I don't find that anywhere in the SharingDialog code, but maybe I'm looking in the wrong place? The types are found are: 
https://github.com/dhis2/ui/blob/master/components/sharing-dialog/src/sharing-dialog.js#L194

Reply:
Yes, I need to merge this: https://github.com/dhis2/ui/pull/1007

Reply:
Ah, I see. Is there any way to export those type constants so they can be used here rather than using a string? `type={INTERPRETATION}`